pallets/registry/src/types.rs

```diff
@@ -69,30 +66,28 @@ pub struct CompleteProof<Hash> {
 }
 
 /// Generates the leaf hash from underlying data, other hashes remain the same.
-///
-/// The Keccak is the hasher compatible with Ethereum.
 impl From<CompleteProof<H256>> for proofs::Proof<H256> {
 	fn from(mut proof: CompleteProof<H256>) -> Self {
 		// Generate leaf hash from property ++ value ++ salt
 		proof.property.extend(proof.value);
 		proof.property.extend(&proof.salt);
-		let leaf_hash = sp_io::hashing::keccak_256(&proof.property).into();
+		let leaf_hash = blake2_256(&proof.property).into();
```

Comment:
Where/how is this hash consumed? Do other bits of code need to be updated for the changed algorithm?

Reply:
no just the proof verification that will be affected. completely independent

Reply:
more context: we dont use kecckak excpet when we are bridging to eth. Only code that bridge to eth is BundledHasher, which we are going to remote soon